jerry-core/ecma/builtin-objects/ecma-builtin-json.c

```diff
@@ -1789,6 +1798,41 @@ ecma_builtin_json_stringify (ecma_value_t this_arg, /**< 'this' argument */
   return ret_value;
 } /* ecma_builtin_json_stringify */
 
+/**
+ * Dispatcher of the built-in's routines
+ *
+ * @return ecma value
+ *         Returned value must be freed with ecma_free_value.
+ */
+ecma_value_t
+ecma_builtin_json_dispatch_routine (uint8_t builtin_routine_id, /**< built-in wide routine identifier */
+                                     ecma_value_t this_arg, /**< 'this' argument value */
+                                     const ecma_value_t arguments_list_p[], /**< list of arguments
+                                                                             *   passed to routine */
+                                     uint32_t arguments_number) /**< length of arguments' list */
+{
+  JERRY_UNUSED (this_arg);
+  ecma_value_t arg0 = (arguments_number > 0) ? arguments_list_p[0] : ECMA_VALUE_UNDEFINED;
+  ecma_value_t arg1 = (arguments_number > 1) ? arguments_list_p[1] : ECMA_VALUE_UNDEFINED;
+  ecma_value_t arg2 = (arguments_number > 2) ? arguments_list_p[2] : ECMA_VALUE_UNDEFINED;
```

Comment:
To keep it in sync with the methods (parse, stringify) we should start the naming from 1 (arg0 should be arg1, etc..)